org-roam-capture.el

```diff
@@ -277,25 +277,54 @@ This function is used solely in Org-roam's capture templates: see
     (widen)
     (goto-char (point-max))))
 
+(define-error 'org-roam-malformed-template
+  "Malformed template in `org-roam-capture-templates'"
+  'org-roam-errors)
```

Comment:
I define a new error to make it obvious that it comes from `org-roam-capture-templates`, which wouldn't be the case with a mere `wrong-type`.